java/ql/lib/change-notes/2022-09-22-android-deeplink-flow-steps.md

```diff
@@ -0,0 +1,5 @@
+---
+category: minorAnalysis
+---
+* Added data flow steps for tainted Android intents that are sent to services and receivers.
+* Updated data flow step for tainted Android intents that are sent to activities.
```

Comment:
Maybe we should mention what's been updated. Suggestion:

```suggestion
* Improved the data flow step for tainted Android intents that are sent to activities so that more cases are covered.
```